src/resolve/resolved-dns-query.c

```diff
@@ -127,36 +130,30 @@ static int dns_query_candidate_add_transaction(DnsQueryCandidate *c, DnsResource
 }
 
 static int dns_query_candidate_go(DnsQueryCandidate *c) {
+        _cleanup_(dns_query_candidate_unrefp) DnsQueryCandidate *keep_c;
```

Comment:
i'd really initialize this to zero here, even if we override it right after. having non-initialized `_cleanup_` variables have the smell of being a bug...

Reply:
Ack.